### ISSUE TITLE
Domibus

### ISSUE THREAD

**Issue by bhack** [FIWARE/helm-charts#78]
As Domibus was added again as third party in the Fiware catalog do you plan to add a Domibus chart?

I found an helm chart  example in:
https://github.com/dladlk/delis/tree/dev/delis-test-server/helm

**Comment by wistefan** [comment]
Hi @bhack ,
we don't have such plans, yet. But we are open for contributions, so a PR would be welcome:)

**Comment by bhack** [comment]
As @dladlk has done the majority of the work there it could be interesting if they are interested to "upstream" it here and on https://artifacthub.io/

**Comment by bhack** [comment]
> The main challenge with Helm was to have some convenient way to customize domibus.properties

Yes It is not a challenge for HELM only but also for a more traditional deployment as you need to maintain patching scripts over the default conf.
